### DISCUSSION TITLE
Provider yml - how to turn off the sorting (ORDER BY)

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#1078] [Configuration] [answered]
I have a table with over 1.2 million records and want to turn off the sorting when displaying the Collection. The response time is too long for each page.

Is it possible?
![observations](https://github.com/interactive-instruments/ldproxy/assets/1905113/2f31382c-8cd3-415b-9da4-1fa831b591e8)

**Comment by cportele** [comment, marked as answer]
For large collections the expensive part is the determination of "numberMatched", not the use of ORDER BY. You can switch this off by setting [`computeNumberMatched`](https://docs.ldproxy.net/providers/feature/sql.html#query-generation) to `false`.

Add the following to the provider configuration:

```yaml
queryGeneration:
  computeNumberMatched: false
```

Does this help?

**Reply by Sverre-Iversen** [reply]
>Does this help?
Yes it worked. Thanks 👍
